### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheHasagi**
* New improvements 
* New logo added
* Added contributing column with anchors
* New shields were added with release & beta versions (clickable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,24 +46,38 @@ To get more information and to download AdGuard for Android, visit our website [
...
-* Latest nightly version available [here](https://agrd.io/android_nightly)
+<a id="contribution-other"></a>
+### Other options
 
-You can find all AdGuard for Android releases here: https://github.com/AdguardTeam/AdguardForAndroid/releases
+Here is a [dedicated page](https://adguard.com/contribute.html) for those who are willing to contribute.
 
+
+<a id="beta-tester"></a>
 ### How to become a beta tester?
```
Comment (ameshkov) [inline README.md]: Twice about beta testing?

Comment (TheHasagi) [reply README.md]: New commig is ready :)

File: README.md [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+  * [How to report an issue](#issue)
+  * [Translating AdGuard](#contribution-translating)
+  * [Other options](#contribution-other)
+   * [How to become a Beta tester](#beta-tester)
+   * [Our plans](#our-plans)
+
+### Disclaimer
+
+> Disclaimer
+> AdGuard for Windows is not an open source project. We use GitHub as an open bug tracker for users to see what developers are working on. However, we at AdGuard create [a lot of open source software](https://github.com/search?o=desc&q=topic%3Aopen-source+org%3AAdguardTeam+fork%3Atrue&s=stars&type=Repositories).
```
Comment (vbagirov) [inline README.md]: Windows

File: README.md [hunk 3]
```diff
@@ -1,7 +1,41 @@
...
+    <a href="https://github.com/AdguardTeam/AdguardForAndroid/releases/tag/v3.1.0-beta">
+    <img src="https://img.shields.io/badge/beta-v3.1.0-yellow.svg" alt="Beta version>" />
+    </a>
+
+
+* [Contribution](#contribution)
+  * [How to report an issue](#issue)
+  * [Translating AdGuard](#contribution-translating)
+  * [Other options](#contribution-other)
+   * [How to become a Beta tester](#beta-tester)
```
Comment (vbagirov) [inline README.md]: 30 и 31 строки должны быть на том же уровне что и 26, сейчас они на уровень ниже

File: README.md (hunk 1 above)
Comment (vbagirov) [reply README.md]: You need to use ## instead of ### here.

File: README.md [hunk 4]
```diff
@@ -11,24 +45,33 @@ To get more information and to download AdGuard for Android, visit our website [
...
 
-You can find all AdGuard for Android releases here: https://github.com/AdguardTeam/AdguardForAndroid/releases
 
+<a id="beta-tester"></a>
 ### How to become a beta tester?
 
 You can submit an application to participate in AdGuard for Android beta-testing program. All necessary information on this topic can be found on the [dedicated page](https://adguard.com/beta.html).
+
+<a id="our-plans"></a>
+### Our plans
```
Comment (vbagirov) [inline README.md]: You need to use ## instead of ### here.

File: README.md [hunk 5]
```diff
@@ -11,24 +45,33 @@ To get more information and to download AdGuard for Android, visit our website [
...
 
+<a id="contribution"></a>
+## Contribution
+
+<a id="issue"></a>
 ### How to report an issue?
 
 GitHub can be used to report a bug or to submit a feature request. To do so, go to [this page](https://github.com/AdguardTeam/AdguardForAndroid/issues) and click the *New issue* button.
 
 >**Note:** for the filter-related issues (missed ads, false positives etc.) use the [dedicated repository](https://github.com/AdguardTeam/AdguardFilters).
```
Comment (vbagirov) [inline README.md]: Здесь наверное лучше ссылку на https://reports.adguard.com/new_issue.html

File: README.md (hunk 2 above)
Comment (TheHasagi) [reply README.md]: *

File: README.md (hunk 3 above)
Comment (TheHasagi) [reply README.md]: *

File: README.md (hunk 1 above)
Comment (TheHasagi) [reply README.md]: *

File: README.md (hunk 4 above)
Comment (TheHasagi) [reply README.md]: *

File: README.md (hunk 5 above)
Comment (TheHasagi) [reply README.md]: *

**Pull request merged**